ios/FluentUI/People Picker/ContactCollectionView.swift

```diff
@@ -117,47 +165,67 @@ extension ContactCollectionView: ContactViewDelegate {
         if let contactCollectionViewDelegate = contactCollectionViewDelegate, let currentTappedIndex = contactViewToIndexMap[contact] {
             let indexPath = IndexPath(item: currentTappedIndex, section: 0)
             scrollToContact(at: indexPath)
-            contactCollectionViewDelegate.didTapOnContactViewAtIndex?(index: currentTappedIndex, personaData: contactList[currentTappedIndex])
+            contactCollectionViewDelegate.didTapOnContactViewAtIndex?(index: currentTappedIndex, personaData: personas[currentTappedIndex])
         }
     }
 }
 
 extension ContactCollectionView: UICollectionViewDataSource {
     @objc public func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
-        return contactList.count
+        return personas.count
     }
 
     @objc public func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
         let cell = collectionView.dequeueReusableCell(withReuseIdentifier: ContactCollectionViewCell.identifier, for: indexPath) as! ContactCollectionViewCell
-        cell.setup(contact: contactList[indexPath.item])
-        cell.contactView.contactViewDelegate = self
-        contactViewToIndexMap[cell.contactView] = indexPath.item
+        cell.setup(contact: personas[indexPath.item], size: size.contactViewSize)
+
+        if contactCollectionViewDelegate != nil {
+            cell.contactView!.contactViewDelegate = self
+        }
+
+        contactViewToIndexMap[cell.contactView!] = indexPath.item
 
         return cell
     }
 }
 
 extension UIContentSizeCategory {
-    var contactHeight: CGFloat {
+    func contactHeight(size: ContactView.Size) -> CGFloat {
+        var height: CGFloat = 0
+
         switch self {
         case .extraSmall:
-            return 115.0
+            height = 117
         case .small:
-            return 117.0
+            height = 119
         case .medium:
-            return 118.0
+            height = 120
         case .large:
-            return 121.0
+            height = 123
         case .extraLarge:
-            return 125.0
+            height = 127
         case .extraExtraLarge:
-            return 129.0
-        case .extraExtraExtraLarge, .accessibilityMedium,
-             .accessibilityLarge, .accessibilityExtraLarge,
-             .accessibilityExtraExtraLarge, .accessibilityExtraExtraExtraLarge:
-            return 135.0
+            height = 131
+        case .extraExtraExtraLarge:
+            height = 137
+        case .accessibilityMedium:
+            height = 141
+        case .accessibilityLarge:
+            height = 151
+        case .accessibilityExtraLarge:
+            height = 165
+        case .accessibilityExtraExtraLarge:
+            height = 178
+        case .accessibilityExtraExtraExtraLarge:
+            height = 194
         default:
-            return 135.0
+            break
         }
+
+        if size == .small {
```

Comment:
What's the reason behind this logic? I'm wondering why the small size subtracts a fixed constant even though the heights are variable.
Maybe worth a comment explaining it?

Reply:
The difference is height is constant since we remove a label for the smaller version.

Reply:
Got it. I think it's worth a comment explaining it and/or moving the 38 value to a named constant (subtitleLabelHeight for example).

Reply:
Will add a comment. As far as a named constraint, the idea here was to merge all these constants into one big contactHeight method.

Reply:
Just to be explicit on the reason why it's taking out from the height only on the smaller size.
I think this should be more like: "// remove/compensate the hight of the subtitle label that is not added/used in the smaller size".